spec/lib/danger/ci_sources/bitrise_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "danger/ci_source/xcode_server"
+
+describe Danger::XcodeServer do
```

Comment:
If you change this, and the line above to use the bitrise, then the tests should start to work.

You're doing great so far 👍